app/services/create_proposed_organisation_edit.rb

```diff
@@ -21,7 +21,8 @@ def initialize(listener, params, model_klass, user_klass, mailer_klass)
   def run
     unless editor.siteadmin?
       merge_in_non_published_fields
-      send_email_to_superadmin_about_org_edit
+      #send_email_to_superadmin_about_org_edit
+      deliver_edit_org_waiting_for_approval
```

Comment:
so this won't work because is needs arguments passed in.  You've defined it as: `deliver_edit_org_waiting_for_approval(organisation, superadmin_emails)`